api/src/services/observation-service.ts

```diff
@@ -615,13 +626,11 @@ export class ObservationService extends DBService {
 
     // Merge all the table rows into an array of InsertUpdateObservations[]
     const newRowData: InsertUpdateObservations[] = worksheetRowObjects.map((row) => {
-      // TODO: This observationSubcountSignId logic is specifically catered to the observation_subcount_signs code set,
-      // as it is the only code set currently being used in the observation CSVs, and is required. This logic will need
-      // to be updated to be more generic if other code sets are used in the future, or if they can be nullable.
-      const observationSubcountSignId = codeTypeDefinitions.OBSERVATION_SUBCOUNT_SIGN.find(
-        (option) =>
-          option.name.toLowerCase() === getColumnCellValue(row, 'OBSERVATION_SUBCOUNT_SIGN')?.cell?.toLowerCase()
-      )?.id;
+      const observationSubcountSignId = this._getCodeIdFromCellValue(
+        getColumnCellValue(row, 'OBSERVATION_SUBCOUNT_SIGN').cell,
+        codeTypeDefinitions.OBSERVATION_SUBCOUNT_SIGN,
+        'direct sighting'
```

Comment:
just checking: this is meant to be hard coded?  If it is, I would move this to a const at the top of the file. Best practice anytime you have seemingly random numbers or strings in the middle of some code.

Reply:
FIXED